hazelcast/test/src/HazelcastTests8.cpp

```diff
@@ -1763,6 +1764,99 @@ namespace hazelcast {
                 ASSERT_OPEN_EVENTUALLY(leave);
                 ASSERT_OPEN_EVENTUALLY(join);
             }
+
+            class connection_manager_translate : public ClientTestSupport {
+            public:
+                static const address private_address;
+                static const address public_address;
+
+                class test_address_provider : public connection::AddressProvider {
+                    public:
+                        test_address_provider(bool shouldTranslate) : should_translate(shouldTranslate) {}
```

Comment:
```suggestion
                        test_address_provider(bool should_translate) : should_translate_(should_translate) {}
```

None of the tests below contruct this class with should_translate as false BTW.

Reply:
Removed the unused parameter. It was initially there to include more test cases but I later on decided that i will not have those test cases.